src/test/java/emissary/client/HTTPConnectionFactoryTest.java

```diff
@@ -74,6 +76,20 @@ void loadPWFromFile() throws Exception {
         assertNotSame(SSLContext.getDefault(), fromConfig);
     }
 
+    /**
+     * Read a known environment variable configured during setup {@link UnitTest#setupSystemProperties()}
+     *
+     * @throws Exception thrown when an error occurs
+     */
+    @Test
+    void loadPWFromEnv() throws Exception {
+        char[] pw = HTTPConnectionFactory.loadPW("${PROJECT_BASE}");
```

Comment:
Should this test both expected patterns?

Reply:
> Should this test both expected patterns?

Removed the second pattern. We can update the regex in the future if necessary.